Codigo/InvUsuario.bas

```diff
@@ -3472,8 +3471,8 @@ ItemNewbie_Err:
 102     Call TraceError(Err.Number, Err.Description, "InvUsuario.ItemNewbie", Erl)
 End Function
 
-Public Function IsItemInCooldown(ByRef obj As t_UserOBJ) As Boolean
+Public Function IsItemInCooldown(ByVal UserIndex, ByRef obj As t_UserOBJ) As Boolean
```

Comment:
`ByVal UserIndex` can be replace by `byref user as t_User`, no need to pass userindex at all